capa/main.py

```diff
@@ -973,6 +983,9 @@ def main(argv=None):
         format = "freeze"
         with open(args.sample, "rb") as f:
             extractor = capa.features.freeze.load(f.read())
+    elif is_dotnet:
+        # TODO extractor = capa.features.extractors.dotnet.extractor.DnFeatureExtractor(...)
```

Comment:
do we want to move this to https://github.com/mandiant/capa/blob/57fe1e27b60fdd0993f2395c4a52a1dabd6e4794/capa/main.py#L426??

Reply:
yes, good idea, should we add a new `format` (just used internally or providable by the user also?)?